cache/redis_cache.go

```diff
@@ -22,10 +23,16 @@ type redisCache struct {
 	expire time.Duration
 }
 
-const getTimeout = 1 * time.Second
-const putTimeout = 10 * time.Second // the put is long enough for very large cached result (+200MB) because it's also linked to the speed of the reader
+const getTimeout = 2 * time.Second
+const removeTimeout = 3 * time.Second
+const renameTimeout = 2 * time.Second
+const putTimeout = 2 * time.Second
```

Comment:
Those timeouts are weird.. Why remove is bigger and rename is equal to put?

Reply:
good point, I'll rework them: 2 secs for each and we will modify them if we see some issues in CS production. Does it sounds good?